h/search/docs/annotation.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+from elasticsearch_dsl import (Boolean,
+                               DocType,
+                               Date,
+                               field,
+                               Field,
+                               Keyword,
+                               Text,
+                               )
+
+from h.search.target import Target
+
+
+class Annotation(DocType):
+    annotator_schema_version = Text()
```

Comment:
I'm not 100% sure all of these are correct-need to intensely test to verify but I'm more interested in how everyone feels about the general architecture here. @seanh @lyzadanger @robertknight @sheetaluk